starboard/nplb/player_test_util.h

```diff
@@ -29,10 +29,12 @@ namespace nplb {
 
 typedef std::tuple<const char* /* audio_filename */,
                    const char* /* video_filename */,
-                   SbPlayerOutputMode /* output_mode */>
+                   SbPlayerOutputMode /* output_mode */,
+                   const char* /* key_system */>
```

Comment:
std::string, as `key_system` cannot be nullptr?  We also shouldn't keep adding more elements to this tuple, and turn it into a struct, feel free to add a TODO (and create a bug) so we can clean up in future.

Reply:
b/283002236 is created.